doc/frontend.md

```diff
@@ -7,14 +7,12 @@ interface for superusers to manage other users.
 
 ## Contributing
 
-### GOV.UK Frontend
+### Design System
 
-The Frontend app makes use of the
-[GOV.UK Frontend](https://design-system.service.gov.uk/get-started/production/)
-design system to markup and style the web interface.
+The Frontend app makes use of the [GOV.UK Frontend Design System](https://design-system.service.gov.uk/get-started/production/) and the [Ministry of Justice Design System](https://github.com/ministryofjustice/moj-design-system)  to markup and style the web interface.
 
-We install GOV.UK Frontend using npm, to easily track updated releases. The npm
-package is automatically installed as part of the Docker image build.
+We install the design systems using npm, to easily track updated releases. The npm
```

Comment:
We install the MOJ Frontend npm module, not the Design System (which is more about documentation than anything else). GOV.UK Frontend is a dependency of MOJ Frotnend, so maybe we don't need to explicitly mention it.